content/php/concepts/string-functions/terms/stripos/stripos.md

```diff
@@ -0,0 +1,76 @@
+---
+Title: 'stripos()'
+Description: 'Searches for the position of the first occurrence of a string inside another string without regard for case.'
+Subjects:
+  - 'Computer Science'
+  - 'Web Design'
+  - 'Web Development'
+Tags:
+  - 'Strings'
+  - 'Functions'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The **`stripos()`** function performs a case-insensitive search for the position of the first occurrence of a string inside another string. If the string is found, it will return the index of the beginning of the string. If the string isn't found, the function won't return any output.
+
+`stripos()` is binary-safe, meaning it will interpret the string with the correct amount of characters it contains, even if the string contains non-ASCII bytes or null bytes.
+
+## Syntax
+
+```pseudo
+stripos(given_string, string_to_find, start_search)
+```
+
+Required Parameters:
+
+- `given_string` is the string that the function searches.
+- `string_to_find` is the string to search within `given_string`.
+
+Optional Parameters:
+
+- `start_search` tells the function to start the search at a specified index.
+
+## Example
+
+### With Required Parameters
+
+```php
+<?php
+echo stripos("I will return the position of a string", "will");
+?>
+```
+
+> **Note:** String positions start at index 0.
+
+The output of the above code will be `2`, as the string "will" starts at index 2.
+
+`stripos()` is case-insensitive. In the example above, the result would be the same if the second parameter was written as "WILL" instead of "will".
+
+### With Required and Optional Parameters
+
+```php
+<?php
+echo stripos("I will return the position of a string", "will", 3);
+?>
+```
+
+The above code won't return anything, as the search begins at index 3. The string "will" doesn't appear again after index 2.
+
+> **Note:** The output will not return anything, but will evaluate to `false`.
+
+## Codebyte
```

Comment:
Ideally each example in an entry is demonstrating some new functionality. In other words, we strive to add value with each example and not include examples that just duplicate functionality shown elsewhere in the entry. I would suggest making the second example the codebyte and cutting the 'pizza' example, which is redundant.

Reply:
Ok, noted. I have removed the 'pizza' codebyte example and made the second example the codebyte instead. Is it ok to have an example that doesn't return anything in the output when the user runs the codebyte?  If not, I can change the index to start the search, or the string to find, so there will be an output.

Reply:
Yes, I think it's preferable to have an output. I've modified the example further to provide users with some options.